src/modules/feature-modules/innovator/pages/innovation/record/section-edit.component.ts

```diff
@@ -120,6 +120,13 @@ export class InnovationSectionEditComponent extends CoreComponent implements OnI
 
     const formData = this.formEngineComponent?.getFormValues();
 
+    Object.keys(formData?.data || {}).forEach(key => {
```

Comment:
@eduarda-meirinhos-bjss we have a function that makes the validation if is empty (search for `UtilsHelper.isEmpty`).